packages/notifi-react-card/README.md

```diff
@@ -269,7 +269,9 @@ export const Notifi: React.FC = () => {
           nonce: `${nonce}`,
         });
 
-        return result.signature;
+        if (typeof result === 'string') {
```

Comment:
ohhh so can use either or result or result.signature? is that based off the provider?

Reply:
Sorry that I missed this!

No, there are some wallet adapters that return the result as the signature as a `string` while others as a payload object with signature as an object key / value.

The idea is if the wallet adapter returns string, then we can return the result (which is the signature). If the wallet adapter returns a payload type, then we need to signature value. 

Does that help?